### ISSUE TITLE
some problems about training.

### ISSUE THREAD

**Issue by Chromer163** [MarekKowalski/DeepAlignmentNetwork#25]
I wonder how many epochs should be set to train the model, and should the learning rate be updated during training?

**Comment by MarekKowalski** [comment]
Hi,

I don't know the specific number of epochs, I would train the first stage until the validation stops improving at a reasonable rate (like a new best error value once every hour or two).
The learning rate was never changed.

Marek
